### DISCUSSION TITLE
Welcome to translation Discussions!

### DISCUSSION THREAD

**Discussion by kkweon** [ultimate-go-korean/translation#91] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by nicewook** [comment]
Issues 탭에 섞여 들어가있던 토픽들을 분리해 낸 것으로 보면 되겠지요?
